### DISCUSSION TITLE
Is it possible to set the deafult sampling steps and method?

### DISCUSSION THREAD

**Discussion by Lumpsack** [vladmandic/sdnext#943] [Q&A] [answered]
Hi,

Probably something I've missed in settings, but is it possible to set these defaults and I always end up changing them.

Ta.

**Comment by vladmandic** [comment, marked as answer]
modify values in `ui-config.json`

**Comment by Lumpsack** [comment]
Outstanding, thanks again.
For anyone else the specific values rea:
    "txt2img/Sampling method/value": "Euler a",
    "txt2img/Sampling steps/value": 25,
